### ISSUE TITLE
AUC keeps 0.5 when training on taobao datasets

### ISSUE THREAD

**Issue by Woolseyyy** [alimamarankgroup/HPMN#5]
When training DIEN on taobao datasets with the train.py script, I observed that AUC_TEST keeps 0.5 in many hyperparameters settings, e.g., 1e-3 learning rate & 1e-3 reg_lambdas, 1e-3 learning rate & 1e-4 reg_lambdas. The AUC_TEST became normal when the reg_lambda became 1e-5.
Is this normal?  Is this consistent with your experiment?

**Comment by aHuiWang** [comment]
I seem to have encountered the same problem as you, did you solve it?
